grpc/testing/benchmark_service.proto

```diff
@@ -0,0 +1,44 @@
+// Copyright 2015 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// An integration test service that covers all the method signature permutations
+// of unary/streaming requests/responses.
+syntax = "proto3";
+
+import "grpc/testing/messages.proto";
+
+package grpc.testing;
+
+service BenchmarkService {
+  // One request followed by one response.
+  // The server returns the client payload as-is.
+  rpc UnaryCall(SimpleRequest) returns (SimpleResponse);
```

Comment:
It's pretty annoying that SimpleRequest/SimpleResponse aren't included in this file. It makes it less legible.

Reply:
SimpleRequest/SimpleResponse are shared with the interop services, so they need their own proto file.

Reply:
I see. These proto files were weird to begin with, as all the messages used by ReportQpsScenarioService and WorkerService only used in that one service, but the messages exist in a different file. That's probably because this is a pretty old file, like test.proto, back before we had our infrastructure worked out to have messages in the same file.

Sight... report_qps_scenario_service.proto is such a useless file.